Dockerfile

```diff
@@ -1,7 +1,8 @@
-FROM mirror.gcr.io/library/golang:latest
+FROM --platform=$BUILDPLATFORM mirror.gcr.io/library/golang:latest
 COPY . /go/src/github.com/chmouel/gosmee
 WORKDIR /go/src/github.com/chmouel/gosmee
-RUN CGO_ENABLED=0 GOOS=linux go build -a  -ldflags="-s -w"  -installsuffix cgo -o gosmee .
+ARG TARGETARCH 
```

Comment:
would that work on release from the github actions workflow ?

Reply:
I mean it's here how we build images on releases https://github.com/chmouel/gosmee/blob/main/.github/workflows/build.yml#L33

Reply:
No, i will add multi arch  changes to build from github actions.

Reply:
@chmouel i have added multi-arch changes to github actions workflow 
would you please check now,  https://github.com/chmouel/gosmee/pull/20/files#diff-5c3fa597431eda03ac3339ae6bf7f05e1a50d6fc7333679ec38e21b337cb6721

Reply:
i have built and pushed image my ghcr.io registry. here is job id: https://github.com/Basavaraju-G/gosmee/actions/runs/3180322175/jobs/5183757367 
and multi-arch gosmee pkg
https://github.com/Basavaraju-G/gosmee/pkgs/container/gosmee